test/espnet2/enh/test_espnet_model.py

```diff
@@ -192,23 +165,19 @@ def test_single_channel_model(
 )
 
 
+pit_wrapper = PITSolver(criterion=FrequencyDomainMSE(compute_on_mask=True))
+
+
 @pytest.mark.parametrize("training", [True, False])
 @pytest.mark.parametrize("mask_type", ["IBM", "IRM", "IAM", "PSM", "PSM^2"])
 @pytest.mark.parametrize(
     "loss_type", ["mask_mse", "magnitude", "spectrum", "spectrum_log"]
```

Comment:
Not effective in current version. Different loss types are not tested.

Reply:
Add unit tests for criterions and wrappers, different loss types are now tested in thier unit tests.